test/TestServerProjects/paging/Generated/PagingClient.cs

```diff
@@ -458,6 +458,82 @@ Page<Product> NextPageFunc(string nextLink, int? pageSizeHint)
             return PageableHelpers.CreateEnumerable(FirstPageFunc, NextPageFunc);
         }
 
+        /// <summary> Define `filter` as a query param for all calls. However, the returned next link will also include the `filter` as part of it. Make sure you don&apos;t end up duplicating the `filter` param in the url sent. </summary>
+        /// <param name="filter"> OData filter options. Pass in &apos;foo&apos;. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual AsyncPageable<Product> DuplicateParamsAsync(string filter = null, CancellationToken cancellationToken = default)
```

Comment:
Can we add a test case for this newly added method?

Reply:
Done